### PR TITLE
test: add syncer heartbeat test

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
test syncer heartbeat in integration test

### What is changed and how it works?

1. turn on `enable-heartbeat` switch
2. make heartbeat interval configurable

### Check List 

Tests 

 - Integration test

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @GregoryIan @csuzhangxc

File: dm/config/subtask.go [hunk 1]
```diff
@@ -93,15 +93,17 @@ type SubTaskConfig struct {
...
-	Meta             *Meta  `toml:"meta" json:"meta"`
-	Timezone         string `toml:"timezone" josn:"timezone"`
+	SourceID                string `toml:"source-id" json:"source-id"`
+	ServerID                int    `toml:"server-id" json:"server-id"`
+	Flavor                  string `toml:"flavor" json:"flavor"`
+	MetaSchema              string `toml:"meta-schema" json:"meta-schema"`
+	RemoveMeta              bool   `toml:"remove-meta" json:"remove-meta"`
+	DisableHeartbeat        bool   `toml:"disable-heartbeat" json:"disable-heartbeat"` //  deprecated, use !enable-heartbeat instead
+	HeartbeatUpdateInterval int    `toml:"heartbeat-update-interval" json:"heartbeat-update-interval"`
+	HeartbeatReportInterval int    `toml:"heartbeat-report-interval" json:"heartbeat-report-interval"`
```
Comment (csuzhangxc) [inline dm/config/subtask.go]: where do we set the default value for them?

Comment (amyangfei) [reply dm/config/subtask.go]: https://github.com/pingcap/dm/blob/master/syncer/heartbeat.go#L91
in heartbeat logic

File: tests/all_mode/run.sh [hunk 2]
```diff
@@ -32,6 +32,9 @@ function run() {
 
     # use sync_diff_inspector to check data now!
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+
+    check_metric $WORKER1_PORT 'dm_syncer_replication_lag{task="test"}' 0 1
+    check_metric $WORKER2_PORT 'dm_syncer_replication_lag{task="test"}' 0 1
```
Comment (csuzhangxc) [inline tests/all_mode/run.sh]: check before the task started?
whether needing to check after replicated some binlog events.

Comment (amyangfei) [reply tests/all_mode/run.sh]: This metric is task related, if task has not started, there is no `dm_syncer_replication_lag` for this task, and if task is not in syncer unit, the `dm_syncer_replication_lag` value is the updated timestamp.
Do you mean check metric does not exist before task is started?

Comment (csuzhangxc) [reply tests/all_mode/run.sh]: Are these two lines (L36, L37) checking metrics but the task is not started yet?

I mean, check metric after some data are replicated?

Comment (amyangfei) [reply tests/all_mode/run.sh]: the task starts at L25, L28 ensures load stage is done, L30,L31 sends increment payload in upstream.

Comment (csuzhangxc) [reply tests/all_mode/run.sh]: Oh, my mistake.

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**